README.md

```diff
@@ -6,4 +6,4 @@
 
 Run the below command in your terminal. 
 
-`npm triple`
+`npx triplej`
```

Comment:
If you have the tool `npx` installed run command below:

```suggestion
`npx triplej`
```
Or you can install it globally with 

```
npm install -g triplej
triplej
```

Reply:
Yep, good to show both if anyone is using an old version of npm that doesn't have npx. Updated!